app/_src/kubernetes-ingress-controller/guides/overview.md

```diff
@@ -76,4 +87,8 @@ the {{site.kic_product_name}}:
 - [Using OpenID-connect plugin](/kubernetes-ingress-controller/{{page.kong_version}}/guides/using-oidc-plugin)
   This guide walks through steps necessary to set up OIDC authentication.
 - [Allow Multiple Authentication Plugins](/kubernetes-ingress-controller/{{page.kong_version}}/guides/allowing-multiple-authentication-methods)
-  This guide walks through the steps for configuring multiple authentication options for consumers.
\ No newline at end of file
+  This guide walks through the steps for configuring multiple authentication options for consumers.
+- [Using Gateway Discovery](/kubernetes-ingress-controller/{{page.kong_version}}/guides/using-gateway-discovery)
```

Comment:
I didn't notice it before, but shouldn't we display this section for KIC >= 2.9 only?

Reply:
Thanks for catching this! I've updated this PR